### DISCUSSION TITLE
Installation from Wixtoolset's msi files take lot of time to show the progress of installation.

### DISCUSSION THREAD

**Discussion by HarshSuman82** [wixtoolset/issues#6489] [Questions]
When I am trying to install the application from the MSI file which was created by using the Wix toolset, It takes lots of time to install the application and doesn't display the progress bar in starting it gives feels like the installer is frozen.

Installer Msi file: [https://www.jiocloud.com/s/?t=CfLgQJXHoRCbDVMg&s=a8](url) 

![image](https://user-images.githubusercontent.com/86040662/123416621-6dfcd000-d5d4-11eb-9cec-b8982f08ff6e.png)

**Comment by BMurri** [comment]
Questions include what custom actions are in your MSI file.

Note that a verbose log includes timestamps, and the progress bar is moved by `ACTION` messages, so looking at the timestamps of the activity before the first such message will let you know where the time is being consumed.
